.github/workflows/managed.yml

```diff
@@ -61,8 +61,14 @@ jobs:
           pushd tools && go mod download -x
           popd        && go mod download -x
 
-      - name: Initialize CI environment
-        run: make env-compose-up # the container workdir is /root/go/src/github.com/percona/pmm
+      - name: Launch PMM Server (see docker-compose.yml)
+        run: |
+          # Note: launching the container with --wait fails for an unknown reason. 
```

Comment:
Yeah, I think is because docker-compose exits with a non-zero code when using wait, so the CI environment treats it as an error/failure.

Reply:
Well, it used to work so far, I view it as a CI oddity, which should be temporary.